### ISSUE TITLE
Kotling version error when using plugin

### ISSUE THREAD

**Issue by mikeyphoenix** [LinusU/flutter_web_auth#9]
I get the below error in visual studio when trying to use the plugin

![image](https://user-images.githubusercontent.com/4477856/67940126-bf747200-fbdb-11e9-9c22-2155f3cdbd27.png)

**Comment by Furqankhanzada** [comment]
```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.10 and higher.
The following dependencies do not satisfy the required version:
project ':flutter_web_auth' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5m 18s
Finished with error: Gradle task assembleDebug failed with exit code 1
```

**Comment by Furqankhanzada** [comment]
@mikeyphoenix  here is quick fix.
Go open this file :
![image](https://user-images.githubusercontent.com/2400215/68828424-7afed100-06c7-11ea-95a5-e7743bed51b6.png)
and change these 2 version according to you need and i will fix the issue. on screenshot i changed them to latest. 
<img width="1251" alt="Screenshot 2019-11-14 at 10 14 29 AM" src="https://user-images.githubusercontent.com/2400215/68828513-be593f80-06c7-11ea-9e0e-d9c3a9d15e86.png">

**Comment by takesfive** [comment]
same problem

**Comment by emailsubjekt** [comment]
pubspec  
flutter_web_auth: ^0.1.2
----------------------------------------------------------------------------------
build.gradle 

buildscript {
    ext.kotlin_version = '1.3.50'
    repositories {
        google()
        jcenter()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:3.5.0'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

---------------------------------------------------------------------------------
I get this error 

FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.10 and higher.
The following dependencies do not satisfy the required version:
project ':flutter_web_auth' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
Finished with error: Gradle task assembleDebug failed with exit code 1

**Comment by RizanPSTU** [comment]
Same issue here. Is there any solution ?

**Comment by lucavinci** [comment]
I am not the author of this plugin but I created a pull request so lets hope the maintainer will fix it asap

**Comment by lucavinci** [comment]
> Same issue here. Is there any solution?

The quick fix that @Furqankhanzada suggested works perfectly for the time being until the pr is merged

**Comment by bakhlov** [comment]
the same problem

**Comment by LinusU** [comment]
Should be fixed in 🚢 0.1.3, very sorry for the delay in merging and releasing!
